everyvoice/cli.py

```diff
@@ -14,6 +14,9 @@
 from everyvoice.model.aligner.wav2vec2aligner.aligner.cli import (
     align_single as ctc_segment,
 )
+from everyvoice.model.aligner.wav2vec2aligner.aligner.cli import (
+    extract_segments_from_textgrid as create_intervals,
```

Comment:
I'm really not a fan of renaming these functions as you import them. When it adds clarity, that's one thing, but here I think it add confusion. `ctc_segment_extract` might be less confusing if you really don't want to keep the original name.